airflow/www/views.py

```diff
@@ -3859,6 +3859,9 @@ class VariableModelView(AirflowModelView):
 
     list_template = 'airflow/variable_list.html'
     edit_template = 'airflow/variable_edit.html'
+    show_template = 'airflow/variable_show.html'
```

Comment:
I'm not really sure we need this file given we have a custom widget, and all this file does is include a `style` tag that could be done in the widget template file just as well.

Reply:
Actually, there's another way to do this that doesn't involve changing the template.


```python
    formatters_columns = {
        'val': lamda v: Markup("<pre>%s</pre>", v.val),
    }
```
I think this should be all you need - no custom widget or templates.

Reply:
Turns out we already have a formatter -- so we just need to update that.

Reply:
but FAB's widget have different formatting than what we need for show variable , i.e preserving the format of the string

Reply:
overriding _show of BaseCRUDView to mask secret values.